### PR TITLE
fix(analytics): fix table reordering breaking the table

### PR DESCRIPTION
**Pull request by Yelinz**
replace uikit sortable with ember-sortable

With uikit sortable the table can break when dragging to specific locations
Peek 2023-03-17 10-17

With ember-sortable it works and uses modifieres which is nice and relativly easily testable
Peek 2023-03-17 11-37

### CODE REVIEW

**Comment by Yelinz** [comment]
Not sure why distribution tests are breaking, running them individually locally is passing

File: packages/analytics/addon/components/ca-field-selector-list.hbs [hunk 1]
```diff
@@ -1,8 +1,4 @@
-<table
-  class="uk-table uk-table-divider uk-table-hover"
-  uk-overflow-auto
-  {{did-insert this.setupUIkit}}
->
+<table class="uk-table uk-table-divider uk-table-hover" uk-overflow-auto>
```
Comment (velrest) [inline packages/analytics/addon/components/ca-field-selector-list.hbs:1]: I think `uk-overflow-auto` is a class https://getuikit.com/docs/utility#overflow

**Review by velrest — COMMENTED** [review]
Otherwise :+1:

File: packages/analytics/addon/components/ca-field-selector-list.hbs (hunk 1 above)
Comment (Yelinz) [reply packages/analytics/addon/components/ca-field-selector-list.hbs]: Yes, changed it.
The tests are also breaking on main... something changed in the floating dependencies

**Review by velrest — APPROVED** [review]

**Pull request merged**